Source/FLHookPluginSDK/headers/FLCoreServer.h

```diff
@@ -944,7 +944,7 @@ namespace pub
 		IMPORT  int DockRequest(unsigned int const &, unsigned int const &);
 		IMPORT  int DrainShields(unsigned int);
 		IMPORT  int EnumerateCargo(unsigned int const &, struct pub::CargoEnumerator &);
-		IMPORT  int ExistsAndAlive(unsigned int);
+		IMPORT  int ExistsAndAlive(unsigned int); // returns 0 for alive, -2 for dead
```

Comment:
While not for this PR, a rule of thumb is if you identify what the input parameter is, you should probably name it so other people can identify it. At this point though it might be better to do that on 4.X.